### ISSUE TITLE
Running app in private host

### ISSUE THREAD

**Issue by amjedomar** [dokku/dokku#3887]
In the previous issue ([Running App Locally](https://github.com/dokku/dokku/issues/3886))

I don't mean hosting app in local machine
both of two apps will be hosting in server (digitalocean droplet)

**My question is** in dokku can I host app in private host (localhost)?, can I do something familiar to the following?

- the `php-api`
```
dokku domains:set php-api localhost:3456
``` 

- the `nodejs-api` code
```
// make http request to php-api using its private host (localhost)
got.get('http://localhost:3456/users'));
```

> By **private host**: I mean this app cannot accessed outside the server, it only can accessed inside the server

**Comment by josegonzalez** [comment]
You currently cannot make internal http requests to other apps deployed on Dokku. This will be available via the [new network support](https://github.com/dokku/dokku/pull/3847) that will be included 0.20.x. This will be released when the 0.20.x release is ready (I do not have a date in mind).

Hope that clarifies things.

> Aside: No need to open a separate issue, we could have continued the conversation in the original.

**Comment by amjedomar** [comment]
ok I understood, thanks dokku team
